client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClientConfig.java

```diff
@@ -61,6 +61,9 @@ public class SchemaRegistryClientConfig {
   public static final String MISSING_VERSION_CACHE_TTL_CONFIG = "missing.version.cache.ttl.sec";
   public static final String MISSING_SCHEMA_CACHE_TTL_CONFIG = "missing.schema.cache.ttl.sec";
 
+  // Randomizing the index of the starting URL can help with load balancing if many clients
+  // are using a shared configuration. Default is false
+  public static final String RANDOMIZE_STARTING_URL = "schema.registry.url.randomize";
```

Comment:
You'll want to define this prop in `AbstractKafkaSchemaSerDeConfig` as well, so users can pass into serdes.

See how one of the other props such as `MAX_RETRIES_CONFIG` is redefined